cognite/client/_cognite_client.py

```diff
@@ -87,7 +88,7 @@ def __init__(self, config: ClientConfig | None = None) -> None:
         self.workflows = WorkflowAPI(self._config, self._API_VERSION, self)
         self.units = UnitAPI(self._config, self._API_VERSION, self)
         self.simulators = SimulatorsAPI(self._config, self._API_VERSION, self)
-
+        self.simulators.models = SimulatorModelsAPI(self._config, self._API_VERSION, self)
```

Comment:
this should be initiated inside of `class SimulatorsAPI`, and it seems like it already is